src/components/Form/FormFactory.js

```diff
@@ -225,8 +253,9 @@ const getFormGroups = ({ fields, fieldsValues, onFormChange, textPosition, label
             'kubevirt-form-group--no-help': !horizontal && !hasValidationMessage,
           })}
         >
-          {label}
+          {!showLabels && label}
```

Comment:
I think the naming is a bit unfortunate here, because we show the label even if we say `!showLabels`

Reply:
I didn't end up needing it.